### PR TITLE
Ed metrics frontend state

### PR DESCRIPTION
**Pull request by gtrufitt**
metrics-updating

Adds the initial state shape and management and API requesting hooking into filters. 

@susiecoleman @shaundillon Would be good to get your opinion on the redux stuff here.

Notes:
- Right now it waits until all responses until updating, this visually is nicer but means one failure or one slow request means all charts fail to update. This is bad architecture so should be changed but will do in a later PR.
- I'll update the webpack config in the base project for the aliases
- There could definitely be a bit more tidying up / componentising of charts and selects in later PRs
- No waiting or failure states yet!

### CODE REVIEW

File: public/js/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import fetch from 'unfetch';
```
Comment (shaundillon) [inline public/js/actions/index.js]: I'd call this file what the exported function does, just for avoidance of doubt when including it.

File: public/js/components/Filters/Filters.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {Grid, Row, Col} from 'react-flexbox-grid';
+
+const Filters = ({onSelectChange, filterVals}) => {
```
Comment (shaundillon) [inline public/js/components/Filters/Filters.js]: I don't think the curly brackets are needed here

File: public/js/components/Filters/Filters.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    return (
+        <form className={'form'}>
```
Comment (shaundillon) [inline public/js/components/Filters/Filters.js]: Or here

File: public/js/actions/index.js (hunk 1 above)
Comment (shaundillon) [inline public/js/actions/index.js]: Also, is fetch used here?

File: public/js/services/getChartData.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export default function getChartData(filterVals) {
```
Comment (shaundillon) [inline public/js/services/getChartData.js]: I believe fetch needs importing here

File: public/js/actions/index.js (hunk 1 above)
Comment (gtrufitt) [inline public/js/actions/index.js]: It's called... `fetch` :/ https://github.com/developit/unfetch/blob/master/src/index.js https://github.com/developit/unfetch#usage

File: public/js/components/Filters/Filters.js (hunk 2 above)
Comment (gtrufitt) [inline public/js/components/Filters/Filters.js]: They're needed because I'm destructuring the `props` object into those vars

**Comment by gtrufitt** [comment]
Thanks Shaun!

**Pull request merged**